package-scripts.js

```diff
@@ -299,8 +299,26 @@ module.exports = {
         }
       },
       watch: {
-        script: 'nps docs.preprocess && eleventy --serve',
-        description: 'Watch docs for changes & build'
+        default: {
+          script:
+            'nps docs.watch.build && concurrently "nps docs.watch.rebuild" "nps docs.watch.server"',
+          description: 'Watch for changes & rebuild & serve docs',
+          hiddenFromHelp: true
+        },
+        build: {
+          script: 'eleventy && jsdoc -c jsdoc.conf.json',
+          description: 'Build docs'
+        },
+        rebuild: {
+          script:
+            'chokidar "docs/**/*" -i "docs/_dist" -i "docs/_site" -c "nps docs.watch.build"',
+          description: 'Watch for changes and rebuild docs'
+        },
+        server: {
+          script:
+            'browser-sync start --server docs/_site --files "docs/_site/**/*"',
```

Comment:
Currently `browser-sync` included in the `eleventy` package. So, `eleventy` don't use `browser-sync` in future, this command will be broken.
We should include  `browser-sync` in our `devDependencies`.

Reply:
Thank you for your advice! I installed it for `devDependencies` now.